src/main/java/edu/cornell/kfs/concur/batch/xmlObjects/PdpFeedAccountingEntry.java

```diff
@@ -124,5 +124,13 @@ public KualiDecimal getAmount() {
     public void setAmount(KualiDecimal value) {
         this.amount = value;
     }
+    
+    public String getDebugInfo() {
```

Comment:
Shouldn't this method be named "toString()" to conform to Java standards? 
I saw this method "getDebugInfo()"in other SAE business objects and thought that it was odd it was not the toString function.